layouts/try/baseof.html

```diff
@@ -21,116 +22,26 @@
         .subtitle-col-custom-style {
             margin-right: 20px;
             padding-bottom: 20px;
-        };
+        }
+
+        ;
```

Comment:
no need for a semicolon outside the CSS block here I don't think.  Not sure why there was one before this PR either. 🤔

Reply:
whoops!